dpnp/backend/kernels/dpnp_krnl_random.cpp

```diff
@@ -926,6 +927,75 @@ void dpnp_rng_rayleigh_c(void* result, const _DataType scale, const size_t size)
     }
 }
 
+template <typename _DataType>
+void dpnp_rng_shuffle_c(
+    void* result, const size_t itemsize, const size_t ndim, const size_t high_dim_size, const size_t size)
+{
+    if (!(size) || !(high_dim_size > 1))
```

Comment:
high_dim_size == 1 is not acceptable?

Reply:
No, there is no need shuffle for one elem.